### PR TITLE
W-14029320-aws-update-cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

**Review by ms-jscholten — DISMISSED** [review]

File: rpa-builder/6.4/modules/ROOT/pages/_partials/aws-doc-processing.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// tag::general[]
+Action steps that use Amazon Textract services process the document in a synchronous fashion, therefore the document is not stored by Amazon. These action steps automatically convert PDF files to PNG images. You can adjust the fidelity of the conversion in the properties grid of the action step.
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.4/modules/ROOT/pages/_partials/aws-doc-processing.adoc]: I'd reword the passive to:
"...therefore Amazon does not store the document."

File: rpa-builder/6.4/modules/ROOT/pages/_partials/aws-doc-processing.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// tag::general[]
+Action steps that use Amazon Textract services process the document in a synchronous fashion, therefore the document is not stored by Amazon. These action steps automatically convert PDF files to PNG images. You can adjust the fidelity of the conversion in the properties grid of the action step. 
+// end::general[]
+
+// tag::specific[]
+This action step processes the document in a synchronous fashion, therefore the document is not stored by Amazon, and automatically convert PDF files to PNG images. You can adjust the fidelity of the conversion in the properties grid.
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.4/modules/ROOT/pages/_partials/aws-doc-processing.adoc]: Sorry to say I can't get the meaning of this sentence. Do you mean?:

- This action step processes the document in a synchronous fashion, therefore Amazon does not store the document, and it automatically converts PDF files to PNG images. 
or
- This action step processes the document in a synchronous fashion, therefore Amazon does not store the document. These action steps automatically convert PDF files to PNG images.

Comment (Cristian-Venticinque) [reply rpa-builder/6.4/modules/ROOT/pages/_partials/aws-doc-processing.adoc]: I was re-reading this because it is confusing, what about this: 
_This action step processes the document in a synchronous fashion (Amazon does not store the document) and automatically converts PDF files to PNG images. You can adjust the fidelity of the conversion in the properties grid._

Comment (Crispy-Salesforce) [reply rpa-builder/6.4/modules/ROOT/pages/_partials/aws-doc-processing.adoc]: Yes! This version read well! 👍

**Review by Crispy-Salesforce — APPROVED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]
